ginlsat_test.go

```diff
@@ -0,0 +1,180 @@
+package ginlsat
```

Comment:
Nice testing framework, hadn't heard about it yet.
It would be cool if we used github workflows to automatically run these tests on every push. You can find an example here: https://github.com/getAlby/lndhub.go/blob/main/.github/workflows/integration_tests.yaml